src/e2e-test/java/io/cdap/plugin/common/stepsdesign/CommonStepDefinitions.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright © 2021 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.plugin.common.stepsdesign;
+
+import io.cdap.e2e.pages.actions.CdfStudioActions;
+import io.cdap.e2e.pages.locators.CdfSchemaLocators;
+import io.cdap.e2e.pages.locators.CdfStudioLocators;
+import io.cdap.e2e.utils.CdfHelper;
+import io.cdap.e2e.utils.JsonUtils;
+import io.cdap.e2e.utils.PluginPropertyUtils;
+import io.cdap.e2e.utils.SeleniumHelper;
+import io.cdap.plugin.utils.E2ETestConstants;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import org.junit.Assert;
+import org.openqa.selenium.WebElement;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * GCP Plugin's common stepDesigns.
+ */
+public class CommonStepDefinitions implements CdfHelper {
```

Comment:
I don't find anything specific to google-cloud in these step definitions. Can we move them to framework?

Reply:
Raised PR on e2e framework https://github.com/cdapio/cdap-e2e-tests/pull/64 
Please review.